src/processor/operator/ddl/add_rel_property.cpp

```diff
@@ -6,11 +6,13 @@ namespace kuzu {
 namespace processor {
 
 void AddRelProperty::executeDDLInternal() {
-    AddProperty::executeDDLInternal();
+    defaultValueEvaluator->evaluate();
```

Comment:
This code looks confusing. Why not just call 1 function, e.g., `getDefaultVal` that does evaluate as its first step (or call `evaluateAndGetDefaultValue` or something. Also it looks like you don't need to get the defaultVal as the first thing. You can get it either inlined (if you make it a single function) or right before calling ` StorageUtils::createFileForRelPropertyWithDefaultVal` on line 14 below.

Reply:
Moved the evaluate inside `getDefaultVal`, and inlined the call of `getDefaultVal`.